mupdf-sys/build.rs

```diff
@@ -513,6 +528,20 @@ fn main() {
     println!("cargo:rerun-if-changed=wrapper.h");
     println!("cargo:rerun-if-changed=wrapper.c");
 
+    if let Ok(ref target_os) = env::var("CARGO_CFG_TARGET_OS") {
+        if target_os == "windows" {
+            #[cfg(target_env = "msvc")]
```

Comment:
For cross compiling support, `target_env` should also be obtained from env vars.

Reply:
I'll will make changes for other `msvc` references in `build.rs` as well.